javascript/leftJoin/left.join.test.js

```diff
@@ -0,0 +1,10 @@
+const leftJoin = require('./left-join');
+
+describe('Left Join', () => {
+  it('join two tables', () => {
+    let table1 = [{'fond':'enamored'}, {'wrath':'anger'},{'diligent':'employed'},{'outfit':'garb'},{'guide':'usher'}];
+    let table2 =[{'fond':'averse'}, {'wrath':'delight'},{'diligent':'idle'},{'guide':'follow'},{'flow':'jam'}];
+    let result= leftJoin(table1,table2);
+    expect(result).toBeDefined();
+  });
+});
```

Comment:
Once you have a chance to come back to this and get the test passing I will regrade for you!